.github/workflows/shellcheck.yml

```
@@ -34,11 +34,13 @@ jobs:
             github.com:443
             pkg-containers.githubusercontent.com:443
       - uses: actions/checkout@v2
-      - name: Install shellcheck
+      - name: Add brew to PATH
+        run: eval "$(/home/linuxbrew/.linuxbrew/bin/brew shellenv)"
```

i think the subshell is the issue here - maybe 
```suggestion
        run: /home/linuxbrew/.linuxbrew/bin/brew shellenv && echo "PATH=$PATH" >> $GITHUB_ENV
```
?

---

I'm going to try the `Homebrew/actions/setup-homebrew` action next as its recommended in the [[Ubuntu] homebrew will be removed from $PATH](https://github.com/actions/runner-images/issues/6283) accouchement.